test_scripts/WebEngine/WebEngineProjection/011_Processing_show_rpc.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Processing of Show rpc when application has active WEB_VIEW template
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) WebEngine App with WEB_VIEW HMI type is registered
+-- Steps:
+-- 1) App sends `Show` request without `templateConfiguration` for main window (windowID is not defined)
+-- SDL does:
+--  - proceed with `Show` request successfully
+--  - not send `OnSystemCapabilityUpdated` to App
+---------------------------------------------------------------------------------------------------
+--[[ General test configuration ]]
+config.defaultMobileAdapterType = "WS"
+
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update WS Server Certificate parameters in smartDeviceLink.ini file", common.commentAllCertInIniFile)
+common.Step("Add AppHMIType to preloaded policy table", common.updatePreloadedPT, { appSessionId, { appHMIType }})
+common.Step("Start SDL, HMI, connect WebEngine device", common.start)
```

Comment:
@VjKlepikov @aderiabin Why is some cases we split the start of SDL and HMI and WebEngine device connecting into two steps https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/34/files#diff-68a100323919bd7a65d241ecaa97c733R127-R128 but in some cases (like here) we use one step for it. Let's keep consistency if it's possible

Reply:
@LitvinenkoIra Absolutely agree. It will be done before final review on delivery branch.

Reply:
@aderiabin @LitvinenkoIra  Please note that there is dependence on
   --[[ General test configuration ]] config.defaultMobileAdapterType = "WS"  to receive MAC address for WebEngine device via utils.getDeviceMAC in common.getShowParams.